package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Nits: directories is unnecessary; repository can be replaced with `"repository": "terinjokes/console-spec"`; bugs and homepage can be omitted.

promise-debounce is also not necessary for your workflow.